ext/agent/jobs/error_discovery.go

```diff
@@ -57,7 +61,11 @@ func isContainerInErrorState(state *types.ContainerState) bool {
 }
 
 func areLogsIndicatingErrorOrWarning(logs string) bool {
-	regexWarningError := `(?i)(error|warning|exception|err|warn)`
+	regexWarningError := `(?i)(error|warning|exception|err|warn|critical|fatal|stacktrace|
```

Comment:
I would suggest sorting alphanumerically these values. It will be much faster in the future to verify if all potential errors are included in the regex

Reply:
nice catch it helped with removing undiscovered duplicate